docs/src/pages/components/dialog.mdx

```diff
@@ -0,0 +1,373 @@
+---
+name: dialog
+route: /components/dialog
+---
+
+import Button from '@pluralsight/ps-design-system-button'
+import { layout } from '@pluralsight/ps-design-system-core'
+import Dialog from '@pluralsight/ps-design-system-dialog'
+import * as Text from '@pluralsight/ps-design-system-text'
+
+import { Guideline, Intro, Types, Usage } from '../../components/component'
+import { ContentGridVisual, ModalGuidelineExample } from '../../components/dialog'
+
+# Dialog
+
+<Intro>
+  The purpose of a Dialog is to provide actionable messaging and may appear contextually or as a [modal](#modal). The Dialog adapts to various amounts and types of content. For contextual, non-actionable messaging, consider the [Tooltip](/components/tooltip) instead.
+</Intro>
```

Comment:
`Intro` doesn't use mdx and doesn't render the links

Reply:
Correct.  I think we should make it do so.  Filed this yesterday. 

https://github.com/pluralsight/design-system/issues/1212

Since we're not live, I'm ok merging one before the other, as long as the work is captured.  You?